spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainer.java

```diff
@@ -106,7 +103,7 @@
 	private long receiveTimeout = DEFAULT_RECEIVE_TIMEOUT;
 
 	// Map entry value, when false, signals the consumer to terminate
-	private Map<BlockingQueueConsumer, Boolean> consumers;
+	private Set<BlockingQueueConsumer> activeConsumers;
```

Comment:
Great!
Although the comment on the property is wrong 😀. Plus I don't see reason to rename it. Just consumers is enough.